packages/web/src/lib/wallectconnect-mode.ts

```diff
@@ -201,20 +203,23 @@ export class WallectConnectSession {
 
 	/**
 	 * Execute single transaction or group of transactions (atomic transaction)
-	 * @param execParams transaction parameters or atomic transaction parameters
+	 * @param transactions transaction parameters,  atomic transaction parameters
+	 *  or TransactionAndSign object(SDK transaction object and signer parameters)
 	 */
 	async executeTx(
-		execParams: ExecParams[]
+		transactions: ExecParams[] | TransactionAndSign[]
 	): Promise<algosdk.modelsv2.PendingTransactionResponse> {
 		let signedTxn;
 		let txns: Transaction[] = [];
-		if (execParams.length > 16) {
+		if (transactions.length > 16) {
 			throw new Error("Maximum size of an atomic transfer group is 16");
 		}
-		for (const [_, txn] of execParams.entries()) {
-			txns.push(mkTransaction(txn, await mkTxParams(this.algodClient, txn.payFlags)));
+		if (!isSDKTransactionAndSign(transactions[0])) {
```

Comment:
What if transactions type is `TransactionAndSign`? 
I think we don't need require `sign` field in this case. I think we need few discuss about this problem.